src/NuGet.Core/NuGet.Configuration/PackageSource/NuGetConstants.cs

```diff
@@ -19,5 +19,12 @@ public static class NuGetConstants
 
         
         public static readonly string FeedName = "nuget.org";
+
+        public static readonly string DefaultConfigContent = @"<?xml version=""1.0"" encoding=""utf-8""?>
```

Comment:
Build it with xmldoc instead of hard coding it, and test that it is exactly what you expect it to be